src/app/qgisapp.cpp

```diff
@@ -7093,14 +7093,13 @@ void QgisApp::fieldCalculator()
 
 void QgisApp::attributeTable( QgsAttributeTableFilterModel::FilterMode filter )
 {
-  QgsVectorLayer *vectorLayer = qobject_cast<QgsVectorLayer *>( activeLayer() );
-  if ( !vectorLayer )
+  QgsVectorLayer *myLayer = qobject_cast<QgsVectorLayer *>( activeLayer() );
```

Comment:
I think we're hunting down the my prefix in variable name these days ;)

Reply:
I know, I already fixed it but the I reverted the whole PR. Feel free to fix it.